### ISSUE TITLE
Clickable "hints" to describe each financial ratios

### ISSUE THREAD

**Issue by JoshBreitenbach** [jeffreywallphd/OpenFinAL#124]
A tool that can explain the ratios from the ratio table. Maybe be able to hover over a number and have it explain what that means. Or a small question mark that you can click on that gives a summary of the ratio.

**Comment by jeffreywallphd** [comment]
Non-coding tasks include writing the hint descriptions of each ratio.
